charts/opentelemetry-operator/templates/certmanager.yaml

```
@@ -5,23 +5,23 @@ metadata:
   labels:
     {{- include "opentelemetry-operator.labels" . | nindent 4 }}
     app.kubernetes.io/component: webhook
-  name: {{ template "opentelemetry-operator.name" . }}-serving-cert
+  name: {{ template "opentelemetry-operator.fullname" . }}-serving-cert
   namespace: {{ .Release.Namespace }}
 spec:
   dnsNames:
-    - {{ template "opentelemetry-operator.name" . }}-webhook-service.{{ .Release.Namespace }}.svc
-    - {{ template "opentelemetry-operator.name" . }}-webhook-service.{{ .Release.Namespace }}.svc.cluster.local
+    - {{ template "opentelemetry-operator.fullname" . }}-webhook-service.{{ .Release.Namespace }}.svc
+    - {{ template "opentelemetry-operator.fullname" . }}-webhook-service.{{ .Release.Namespace }}.svc.cluster.local
```

```suggestion
    - {{ template "opentelemetry-operator.fullname" . }}-webhook.{{ .Release.Namespace }}.svc
    - {{ template "opentelemetry-operator.fullname" . }}-webhook.{{ .Release.Namespace }}.svc.cluster.local
```

We changed the names of the webhooks so this has to change to match.